### ISSUE TITLE
Why not use torch.save(model, PATH) to serialize the entire model?

### ISSUE THREAD

**Issue by wnagchenghku** [ucbrise/clipper#566]
Hi, in https://github.com/ucbrise/clipper/blob/develop/clipper_admin/clipper_admin/deployers/pytorch.py#L201, I found clipper serializes the model and weight separately using `cloudpickle` and `torch.save(the_model.state_dict(), PATH)`. I'm curious why does clipper do this serialization separately? From https://pytorch.org/docs/stable/notes/serialization.html, pytorch can actually serialize the entire model in one step. I tested that loading the entire model using torch.load is much much much faster than cloudpickle.

**Comment by rkooo567** [comment]
@simon-mo @withsmilo Any thought?

**Comment by simon-mo** [comment]
We should. Can you submit a PR?

**Comment by rkooo567** [comment]
@simon-mo Yep sounds good. I will wait for the author for a couple days and start if he doesn't respond.

@wnagchenghku Would you want to create a PR for this?
